packages/vite-plugin-cloudflare/src/deploy-config.ts

```diff
@@ -0,0 +1,103 @@
+import assert from 'node:assert';
+import * as fs from 'node:fs';
+import * as path from 'node:path';
+import * as vite from 'vite';
+import type { ResolvedPluginConfig } from './plugin-config';
+
+interface DeployConfig {
+	configPath: string;
+	auxiliaryWorkers: Array<{ configPath: string }>;
+}
+
+function getDeployConfigPath(root: string) {
+	return path.resolve(root, '.wrangler', 'deploy', 'config.json');
+}
+
+export function getConfigPaths(root: string) {
+	const deployConfigPath = getDeployConfigPath(root);
+	const deployConfig = JSON.parse(
```

Comment:
Might be safer to use JSONC or JSON5 parser that can handle JSON comments.

Reply:
I'll leave it for now, as we control the output, but we can change this later if necessary.